apps/s_cb.c

```diff
@@ -1507,7 +1507,7 @@ void print_ca_names(BIO *bio, SSL *s)
     const STACK_OF(X509_NAME) *sk = SSL_get0_peer_CA_list(s);
     int i;
 
-    if (sk == NULL || sk_X509_NAME_num(sk) == 0) {
+    if (!SSL_is_server(s) && (sk == NULL || sk_X509_NAME_num(sk) == 0)) {
```

Comment:
I think this is not quite right. Before we were getting the message "No server certificate CA names sent":

````
No server certificate CA names sent
CIPHER is TLS_AES_256_GCM_SHA384
Secure Renegotiation IS supported
````

Now instead, with this patch, we get "Acceptable server certificate CA names":

````
Acceptable server certificate CA names
CIPHER is TLS_AES_256_GCM_SHA384
Secure Renegotiation IS supported
````

Really we want neither of these in the event that the list is empty.